### PR TITLE
The variables anchored, sharp, edge, density  and can_buckle are now boolean.

### PR DESCRIPTION
**Pull request by Evankhell561**
##  About The Pull Request

helps distinguish variables that have a boolean value from variables with numeric value

## Changelog
:cl:
code: The variables anchored, sharp, edge, density and can_buckle are now boolean.
/:cl:

### CODE REVIEW

**Comment by SagaStars** [comment]
image

File: code/modules/power/sensors/powernet_sensor.dm [hunk 1]
```diff
@@ -167,7 +167,7 @@
 			APC_entry["total_load"] = power_to_text(A.lastused_total)
 			// Hopefully removes those goddamn \improper s which are screwing up the UI
 			var/N = A.area.name
-			if(findtext(N, "�"))
+			if(findtext(N, "�"))
```
Comment (MadmanMartian) [inline code/modules/power/sensors/powernet_sensor.dm:170]: 🤔

Comment (Evankhell561) [reply code/modules/power/sensors/powernet_sensor.dm:170]: no idea
image

**Comment by Evankhell561** [comment]
update:

image

**Comment by Evankhell561** [comment]
update2
image

**Comment by Kurgis** [comment]
Next step, enforced newlines PR, with a total of 1000+ file changes.

**Pull request merged**